envalid.d.ts

```diff
@@ -77,19 +77,26 @@ interface CleanOptions {
 /**
  * Returns a sanitized, immutable environment object.
  * @param environment An object containing your env vars (eg. process.env).
- * @param validators An object that specifies the format of required vars.
  */
-export function cleanEnv(environment: any, validators?: Specs, options?: CleanOptions): any
+export function cleanEnv<T>(environment: any): Readonly<T> & CleanEnv
```

Comment:
Could you add a comment above explaining how this overloading works? Probably just my unfamiliarity with what it's doing, but I don't understand why only one argument is given for this version

Reply:
My intent here was to allow users to explicitly specify the shape of the returned object when no validators are passed in. ...Looking at it a bit closer, though, I think it would work just as well to remove this overload and make the `validators` field on the one below optional. I'll do that now...